CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.32.0 Release
+- Release Date: June 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.19 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.9 has ended and upgrading to server ESR v5.19 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app. See [this blog post](https://mattermost.com/blog/support-for-esr-5-9-has-ended/) for more details.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Breaking Changes
+ - On mobile apps, users will not be able to see group mentions (E20 feature) in the autocomplete dropdown. Users will still receive notifications if they are part of an LDAP group. However, the group mention keyword will not be highlighted.
+ - **Upcoming breaking change** Starting with mobile app v1.33.0 (to be released on July 16th), users on server versions below v5.19 may experience issues with how attachments, link previews, reactions and embed data are displayed.
+ 
+### Highlights
+
+#### Upgrade to React Native 0.62
```

Comment:
Why a highlight? Not much user experience benefit from my perspective

Reply:
But it is from a dev perspective, unless the changelog is almed to end users?

Reply:
Aimed at admins and end users mostly

Reply:
I’ll make my case from an admin point of view then:

RN62 introduced a few very relevant internal changes, as an admin if I know this was introduced in the mobile app I can make sure BlueCedar or Appdome fully support this for fusioning the app so it can then be used in combination with my EMM sdk, etc...

Reply:
Thanks, added a description.